Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -84,3 +84,65 @@ Deploy application
     cd "BlueprintBaseName.1/src/BlueprintBaseName.1"
     dotnet lambda deploy-serverless
 ```
+
+## Lambda Annotations
+This template now uses Lambda Annotations to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
+
+This automatically handles reading parameters from an APIGatewayProxyRequest and returning an APIGatewayProxyResponse.
```

Comment:
Since a user might not be creating a function for API Gateway we should call out how it can be used for non-APIGateway based functions and benefit from the syncing and dependency injections framework.

Reply:
Added an H3 for trimming out APIGateway, and moved the section on trimming out annotations altogether to its own H3.